osu.Game/Storyboards/Drawables/DrawableStoryboard.cs

```diff
@@ -52,11 +44,6 @@ public DrawableStoryboard(Storyboard storyboard)
             Anchor = Anchor.Centre;
             Origin = Anchor.Centre;
 
-            AddInternal(background = new Background
-            {
-                Anchor = Anchor.Centre,
-                Origin = Anchor.Centre,
-            });

```

Comment:
Are you sure about removing this? Why?

Reply:
check the commit message / code. as far as i can tell, it's not implemented correctly and has no reason to exist here.

Reply:
Can you make an issue for this then? Perhaps even with a note of why you believe it's not implemented correctly?

Reply:
There's many other storyboard features not implemented yet, and I'm not sure if we want to add support for them or not. Am I going to have to create issues for each missing thing? I don't think it's necessary at this point.

Reply:
Wait what other storyboard features? As far as I'm aware triggers are the only things that aren't implemented.

Reply:
(and yes that should have an issue thread of its own)

Reply:
Colour events, for instance